lib/tasks/vendor_deps.rake

```diff
@@ -2,55 +2,9 @@ namespace :test_track do
   task :vendor_deps do
     FileUtils.module_eval do
       cd "vendor/gems" do
-        rm_r Dir.glob('*')
-        `git clone --depth=1 git@github.com:Betterment/style-closet.git && rm -rf style-closet/.git`
```

Comment:
Froze from upgrading style-closet anymore. Lots changed, and it didn't seem worth keeping up?

Reply:
seems right to me, not without making updates to the styles